### ISSUE TITLE
Takeover Vundle

### ISSUE THREAD

**Issue by gmarik** [gmarik/Vundle.vim#608]
Hey guys, 

Please comment below if you're be interested in taking over the Vundle project.

I've not been involved and it hurting the project. 
If you're interested in taking the lead - let me know and we'll discuss transition.

**Comment by Shougo** [comment]
Hi, I think you should create the github organization for Vundle like ctrlp.

https://github.com/ctrlpvim

**Comment by Shougo** [comment]
You or other people should post the anounce in vim_dev or vim_use or reddit/vim, ...
Vundle is major plugin manager, I think the supporters are coming.

**Comment by gmarik** [comment]
Thanks @Shougo, will do that!

**Comment by Shougo** [comment]
:-)

**Comment by Shougo** [comment]
I have announced it in my Twitter account.

**Comment by btreeinfinity** [comment]
I'd be willing to take over, and amp it up, but only with community support and input!

**Comment by Shougo** [comment]
Yes, you don't have to fix VimL code :-)

**Comment by gmarik** [comment]
@btreeinfinity awesome! 
Would be nice to have several ppl onboard as it's not sustainable for one person(unless hardcore OSS-er) IMO.

**Comment by dvidsilva** [comment]
can you list the main things a new maintainer/person/thing/ has to do? Maybe there's people that would be interested but are too scared to take over such a project.

**Comment by gmarik** [comment]
@dvidsilva everyone can find something for oneself: take a look at issues and it could be from helping ppl with advice, testing, fixing a bug to implementing a feature…

**Comment by KamranMackey** [comment]
@gmarik I'd be willing to take over Vundle with @btreeinfinity as well. I don't mind! :smile:

**Comment by gmarik** [comment]
@KamranMackey I don't mind, as long as it makes users happy ;)

**Comment by KamranMackey** [comment]
@gmarik Alright. :smiley:

**Comment by Shokodemon** [comment]
Vundle now returns an error when it's handling itself - are we on this branch still? 

Error:

You are not currently on a branch. Please specify which branch you want to merge with                                                   See git-pull(1) for details.

**Comment by Shougo** [comment]
@Shokodemon This is not user Q/A issue.
You should explain the issue in other place.

It seems you just change the branch manually.
You should reset it with `git checkout master` command in the repository.

**Comment by silfverstrom** [comment]
I would also be willing to maintain the project, as long as @btreeinfinity and @KamranMackey doesn't mind the extra help=)

**Comment by ryanoasis** [comment]
I have some interest in lending a small hand :smile:

**Comment by KamranMackey** [comment]
@silfverstrom I wouldn't mind. :smile: @gmarik Is this enough volunteers now?

**Comment by gmarik** [comment]
@KamranMackey yeah looks like it! 

Will migrate the repo to the organization and configure rights for you guys later today.
Thank you!

**Comment by KamranMackey** [comment]
@gmarik You're welcome. :)

**Comment by ryanoasis** [comment]
:+1:

**Comment by gmarik** [comment]
Hey guys! You're now part of the team. 

Post your feedback in case of any questions.

**Comment by ryanoasis** [comment]
@gmarik cool thanks :+1: 

* How involved do you see yourself now going forward (from what I have read I take it not very involved) ?
* Is it up to the team to figure out and decide the next steps?
* It seemed like there were/are more potential volunteers going forward?

**Comment by gmarik** [comment]
@ryanoasis 
1. /cc me if you'd like to have my input, i'll be following along to get the team going
2. Yes
3. Even better! We'll add more people as they decide to join in.

**Comment by gmarik** [comment]
@KamranMackey master is now reset back to cfd3b2d.

Let's first figure out process we gonna work

**Comment by silfverstrom** [comment]
@gmarik @ryanoasis @KamranMackey @btreeinfinity 
I took the liberty to create a gitter room. If you think it is a good idea, we could keep the discussion as to what the next step is in that room. 
https://gitter.im/silfverstrom/vundlevim

**Comment by ryanoasis** [comment]
@silfverstrom :+1: I even had the same thought last night (especially after the notifications :smirk: ) _but_ I think this would be better: https://gitter.im/VundleVim/Vundle.vim/maintainers thoughts?

**Comment by gmarik** [comment]
@silfverstrom there's existing general gitter room https://gitter.im/VundleVim/Vundle.vim 
@ryanoasis separate maintainers room :+1:.

**Comment by KamranMackey** [comment]
@gmarik May I please have my permissions back? I'm sorry for the mindless PR merging. I won't do it again.

**Comment by ryanoasis** [comment]
@KamranMackey :+1:

**Comment by KamranMackey** [comment]
Since @gmarik won't respond, @VundleVim/team may I please have my collaborator permissions back?

**Comment by ryanoasis** [comment]
I don't think we have that power. Did you try the maintainers chat?

**Comment by gmarik** [comment]
@KamranMackey sorry man, permissions will have to wait a bit. In the mean time you could be helpful in other ways like helping people figure out issues and stuff like that.

**Comment by trygveu** [comment]
Hi, I can lend a hand once in a while with responding to issues. There are numerous issues that have been inactive for a long time, and many of these can probably be closed. What is the preferred way to close an issue?

I would like to be able to do this, without adding extra burden on the project maintainers, and give you a bunch of extra notifications top attend.

1. Ping the OP and encourage them to resolve or close the issue.
2. If no response in 1-2 weeks, ping one of the maintainers, and present a set of reasons for closing the issue.

Is that an okay sequence?

**Comment by ryanoasis** [comment]
@trygveu I think this is a really good idea. :+1: 

Maybe require 2 maintainer votes for closing instead of 1?

Also I think it would be helpful for us to have a maintainers page or something in the wiki to keep track and refer to the rules we decide on

**Comment by wsdjeg** [comment]
why do you create a new plugin neobundle, i hope all of the vundle neobundle etc. combined into one

**Comment by ryanoasis** [comment]
@wsdjeg is that question meant for @Shougo ?

**Comment by wsdjeg** [comment]
@randymorris  yes 
So many plug-in management tools that I can not choose

**Comment by vyp** [comment]
@wsdjeg If I recall correctly, gmarik preferred to keep Vundle simple, but Shougo wanted more features, hence another plugin manager. That's the great thing about free software, we are allowed to have more choices if someone is technical enough to hack on it.

If you are having a hard time choosing, I suggest either start with Vundle (this) or vim-plug. And if you find out later that you want more features, try neobundle.

**Comment by wsdjeg** [comment]
yeal i am using neobundle to lock the plugin to a commit

**Comment by trygveu** [comment]
I feel that you are a bit off-topic @wsdjeg. How Vundle relates to other vim package managers is interesting, maybe we can take that in #387 ?

:+1: @vyp, I think you are touching upon an important point here. Vundle is simple and not very complicated, it has a large user-base and it seems like the functionality fits most of them perfectly fine. I believe we should the principle of being simple should guide Vundle in the future as well.

@ryanoasis, about the wiki-page. Should I just draft one and mark it as "work in progress". What should the title be?

**Comment by Shougo** [comment]
> If you are having a hard time choosing, I suggest either start with Vundle (this) or vim-plug. And if you find out later that you want more features, try neobundle.

I think vim-plug is medium complex plugin manager.  It is not simple like Vundle.
Yes, neobundle is most complex plugin manager.

**Comment by wsdjeg** [comment]
complex does not mean not simple,it just means simple + plus.
such as operating system,engineer and the general user use the same system!
general user alse can use neobundle like vundle just to install plugin without other function

**Comment by ryanoasis** [comment]
@trygveu For the wiki good question. I am not really sure. Maybe something like Collaborators or better yet Project Maintainers section?

Also I am one for the philosophy of "perfect is the enemy of good" :smile:

**Comment by trygveu** [comment]
@ryanoasis Here's a start: https://github.com/VundleVim/Vundle.vim/wiki/Project-Maintainers-Section

**Comment by ryanoasis** [comment]
@trygveu That looks really good (especially for a first version)!

**Comment by trygveu** [comment]
@ryanoasis Hi, I discovered I had cluttered the wiki a bit. There is an existing [Development-Workflow](../wiki/Development-Workflow) page that partly described the issue handling. 

I allowed myself to change the new page to [Handling-issues](../wiki/Handling-issues) and linked to it from the existing page. I also added an open point on Testing on the [Development-Workflow](../wiki/Development-Workflow) page related to #619.

I guess we need to have some working tests up and running before we can finish on describing team process.

**Comment by b232wang** [comment]
which is better vundle or spf13?

**Comment by vyp** [comment]
@b232wang they are two different things

**Comment by wsdjeg** [comment]
hi @gmarik
many people ask me how to use this plugin to manager their plugins,all of then use the old chinese post from the internet.i think it's better to add a file name readme/ch_zn.md ,and i can finish it with PR

**Comment by gmarik** [comment]
@wsdjeg please feel free to add a PR!

**Comment by davidlowryduda** [comment]
Is there still a group of core people working on Vundle? If there is, then I will triage some of the issues and pull requests and recommend courses of action.

**Comment by opsroller** [comment]
Im still here, send em along!

**Comment by ryanoasis** [comment]
Still here doing what little I can (mostly triage and merging small things). I was gone for a bit but now back trying to triage and merge a few things.

**Comment by ryanoasis** [comment]
@gmarik I think @davidlowryduda has been fairly active and reached out to me about possibly being added to the Vundle org. As far as I can tell I have no ability to add anyone to the org.
